src/pages/Category/CategoryCard.js

```diff
@@ -0,0 +1,208 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import GlobalStyles from '../../styles/GlobalStyles';
+
+function CategoryCard(props) {
```

Comment:
```suggestion
function CategoryCard({cardData}) {
```
함수형 컴포넌트에서는 props를 인자로 받아올때 바로 구조분해 할당 할 수 있습니다!

Reply:
맞아요! 오늘 세션보면서 뜨끔했어요.ㅎㅎㅎ 수정했습니다! :)